### PR TITLE
Do not run frontend test on Black Box tests tasks

### PR DESCRIPTION
**Pull request by Nolwenn-cadic-sonarsource**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -238,6 +240,12 @@
   </build>
 
   <profiles>
+     <profile>
+        <id>skip-frontend-tests</id>
```
[the remark this answers is not in the document]
Comment (Nolwenn-cadic-sonarsource) [reply pom.xml]: Yes you are right

File: .cirrus.yml [hunk 2]
```diff
@@ -115,6 +115,7 @@ black_box_test_task:
     folder: ${CIRRUS_WORKING_DIR}/.m2/repository
   bbt_script:
     - mvn verify -Pblack-box-tests -Dsonar.runtimeVersion=${SQ_VERSION} -Dorchestrator.configUrl=file:///$CIRRUS_WORKING_DIR/.cirrus/orchestrator.properties
+
```
Comment (zipengwu) [inline .cirrus.yml:118]: not needed :)

**Review by zipengwu — APPROVED** [review]
LGTM

**Comment by philippe-perrin-sonarsource** [comment]
Ping @Nolwenn-cadic-sonarsource @zipengwu Still relevant?

**Pull request merged**